### ISSUE TITLE
Mac mini mid-2010 MD5 update - 10.11.6

### ISSUE THREAD

**Issue by Alex-twn** [Floris497/mac-pixel-clock-patch-V2#121]
Hello Floris497,

Could you please add the following hashes

---- BEGINNING MD5 HASH SUMS ---- version: 10.11.6 build:15G1212

    otool IOKit: 630fd968d68a0255c790e3af60c3dd86
    IOKit: a73dd1ed37da96000958bb781ce4ac15

---- ENDING MD5 HASH SUMS -------

---- BEGINNING MD5 HASH SUMS ---- version: 10.11.6 build:15G1212

    NVDAGK100Hal: 4dd93ae2924eec914b4db08d57c5766f
    
---- ENDING MD5 HASH SUMS -------

I was able to see only the otool hashes already added to your IOKit patcher but I see in some places that the hash is followed by the build number so just in case you need to add the build number as well here

I hope this will help supporting my LG  29UM58 monitor and I will support you for this.

BTW I did not updated yet to OS 10.12 as I want to check first if it can work on OS 10.11.6, in case it works there, would that work as well on OS 10.12?

Thanks a lot for your help.

**Comment by Alex-twn** [comment]
Hi, did you have time to look at this and add those MD5 Hashes?
Thanks.

**Comment by Floris497** [comment]
I'll look at it tomorrow really short on time right now :)

**Comment by Alex-twn** [comment]
Thanks a lot Floris.
Actually I was able to solve the issue by myself one hour ago and my LG Ultrawide monitor is now perfectly supported at 2560x1080, directly on HDMI output (no adapter involved then).
I played with both the patches files and switchresX so I am not sure what really solved the problem in the end.
I am still interested to get the MD5 implementation but when you have time, no rushes :)
Thanks again.
